frontend/src/metabase/home/homepage/actions.ts

```diff
@@ -3,23 +3,23 @@ import { updateSetting } from "metabase/admin/settings/settings";
 
 export const HIDE_DATA = "metabase/home/homepage/HIDE_DATA";
 export const hideData = createThunkAction(HIDE_DATA, function() {
-  return async function(dispatch) {
+  return async function(dispatch: any) {
```

Comment:
Not sure how the project is configured, but it is quite useful to have warnings for type `any`.
It will allow for faster typing once all the types are in.

Reply:
Yeah, `any` is definitely a bad practise, however all our codebase is in JS and it is hard to add all the types once.